gen-web/res/index.html

```diff
@@ -10,43 +10,44 @@
         <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
     </head>
     <body id='container-fluid' class="step-monitor">
-      <aside id='greeting-container-bookend' class="stepHeader step0 step0b">  
```

Comment:
I am leaving review of html, css and js part of `gen-web` to someone more knowledgeable in this field.

Reply:
@JettTech :point_up_2: